src/tabset/tabset.ts

```diff
@@ -81,15 +81,15 @@ export interface NgbTabChangeEvent {
   template: `
     <ul [class]="'nav nav-' + type" role="tablist">
       <li class="nav-item" *ngFor="let tab of tabs">
-        <a [id]="tab.id" class="nav-link" [class.active]="tab.id === activeId" [class.disabled]="tab.disabled" 
-          href (click)="!!select(tab.id)">
+        <a [id]="tab.id" class="nav-link" [class.active]="tab.id === activeId" [class.disabled]="tab.disabled"
```

Comment:
What about `aria-selected`? do you think we should add it too?

https://www.w3.org/TR/wai-aria/states_and_properties#aria-selected